### PR TITLE
feat: telegram notifications

### PR DESCRIPTION
**Pull request by fredrikved**

### CODE REVIEW

**Comment by fredrikved** [comment]
Made a new channel, but can probably reuse the old one if necessary, but would need some permissions from the owner to add a new bot, etc.

File: activities/notifications.go [hunk 1]
```diff
@@ -59,7 +60,19 @@ func (ns notificationServices) SendEmail(email string, message notifications.Tem
...
 }
 
-func (ns notificationServices) SendTelegramMessage(chatID string, message notifications.Template) error {
+func (ns notificationServices) SendTelegramMessage(channelID string, message notifications.Template) error {
+	bot, err := tgbotapi.NewBotAPI(os.Getenv("TELEGRAM_BOT_TOKEN"))
+	if err != nil {
+		return err
+	}
+	switch t := message.(type) {
+	case notifications.SimpleNotification:
```
Comment (andreasgangso) [inline activities/notifications.go:69]: I think the notifications templates stuff you made here was quite clean but here you go away from that.

Maybe extend Template to also have a RenderMarkdown() / RenderTelegramHTML() (or rename the other one to RenderEmailHtml() etc).
For markdown / html in telegram you need to set parse_mode, see https://github.com/go-telegram-bot-api/telegram-bot-api/blob/4126fa611266940425a9dfd37e0c92ba47881718/configs.go#L45C8-L45C8

**Review by andreasgangso — CHANGES_REQUESTED** [review]

**Comment by andreasgangso** [comment]
> Made a new channel, but can probably reuse the old one if necessary, but would need some permissions from the owner to add a new bot, etc.

saw the channel is public. maybe make it private 👍

**Comment by fredrikved** [comment]
> saw the channel is public. maybe make it private 👍

private it has been made

File: activities/notifications.go (hunk 1 above)
Comment (fredrikved) [reply activities/notifications.go]: Updated

File: activities/notifications.go [hunk 2]
```diff
@@ -59,7 +60,21 @@ func (ns notificationServices) SendEmail(email string, message notifications.Tem
 	return err
 }
 
-func (ns notificationServices) SendTelegramMessage(chatID string, message notifications.Template) error {
+func (ns notificationServices) SendTelegramMessage(channelID string, message notifications.Template) error {
+	bot, err := tgbotapi.NewBotAPI(os.Getenv("TELEGRAM_BOT_TOKEN"))
```
[the remark this answers is not in the document]
Comment (andreasgangso) [reply activities/notifications.go:64]: @fredrikved

[the remark this answers is not in the document]
Comment (andreasgangso) [reply activities/notifications.go:64]: otherwise lgtm

**Pull request merged**